--- dsdt.dsl.orig	2022-11-01 14:14:58.488310639 +0800
+++ dsdt.dsl	2022-11-01 14:14:58.488310639 +0800
@@ -18,7 +18,7 @@
  *     Compiler ID      "ACPI"
  *     Compiler Version 0x00040000 (262144)
  */
-DefinitionBlock ("", "DSDT", 1, "XMCC  ", "XMCC2019", 0x00000002)
+DefinitionBlock ("", "DSDT", 1, "XMCC  ", "XMCC2019", 0x00000003)
 {
     External (_SB_.ALIB, MethodObj)    // 2 Arguments
     External (_SB_.APTS, MethodObj)    // 1 Arguments
@@ -4404,7 +4404,6 @@
                             ALBH (0x06, (FPPT * 0x03E8))
                             ALBH (0x07, (SPPT * 0x03E8))
                             ALBH (0x05, (HSPL * 0x03E8))
-                            ALIB (One, 0x05)
                             Return (One)
                         }
 
