--- dsdt.dsl	2022-06-18 16:57:33.388744781 +0800
+++ ../dsdt.dsl	2022-06-18 16:55:52.723229288 +0800
@@ -18,7 +18,7 @@
  *     Compiler ID      "ACPI"
  *     Compiler Version 0x00040000 (262144)
  */
-DefinitionBlock ("", "DSDT", 1, "XMCC  ", "XMCC2113", 0x00000002)
+DefinitionBlock ("", "DSDT", 1, "XMCC  ", "XMCC2113", 0x00000003)
 {
     External (_SB_.ALIB, MethodObj)    // 2 Arguments
     External (_SB_.APTS, MethodObj)    // 1 Arguments
@@ -5093,7 +5093,7 @@
                             0x01,               // Alignment
                             0x01,               // Length
                             )
-                        IRQ (Edge, ActiveLow, Shared, )
+                        Interrupt (ResourceConsumer, Edge, ActiveLow, Shared, , ,)
                             {1}
                     })
                 }
@@ -1586,7 +1586,8 @@ DefinitionBlock ("", "SSDT", 2, "XMCC ", "XMCC2113", 0x00000002)
             CreateDWordField (Local7, 0x10, A009)
             CreateDWordField (Local7, 0x14, A010)
             Local0 = 0x02
-            While ((Local0 < A148))
+            Local6 = A148
+            While ((Local0 < Local6))
             {
                 Local1 = DerefOf (Arg0 [Local0])
                 Local0++
@@ -5473,7 +5473,7 @@ DefinitionBlock ("", "DSDT", 1, "XMCC  ", "XMCC2113", 0x00000003)
                     {
                         If ((DPTC == One))
                         {
-                            Name (PPPB, Buffer (0x12) {})
+                            Name (PPPB, Buffer (0x11) {})
                             CreateWordField (PPPB, Zero, SSZE)
                             CreateByteField (PPPB, 0x02, PMDH)
                             CreateDWordField (PPPB, 0x03, VCLT)
@@ -5481,7 +5481,7 @@ DefinitionBlock ("", "DSDT", 1, "XMCC  ", "XMCC2113", 0x00000003)
                             CreateDWordField (PPPB, 0x08, VMCL)
                             CreateByteField (PPPB, 0x0C, PMDB)
                             CreateDWordField (PPPB, 0x0D, P3TL)
-                            SSZE = 0x12
+                            SSZE = 0x11
                             PMDH = 0x0B
                             VCLT = Arg0
                             PMDI = 0x0C
